src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml

```diff
@@ -2,7 +2,13 @@ defaults:
   - default_train_params
 
 max_epochs: 250
-lr_updates: [100, 150, 200]
+
+lr_updates:
```

Comment:
hydra actually has a builtin function for that:
```lr_updates : ${range:100,200,50}```

Reply:
It's actually not builtin, it requires to add :
`OmegaConf.register_new_resolver("range", lambda *args: np.arange(*args).tolist())`